config/rbac/role.yaml

```diff
@@ -174,3 +174,18 @@ rules:
   - roles
   verbs:
   - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: manager-role
+  namespace: openshift-monitoring
```

Comment:
@raptorsun so are you saying that even though this role has the NS defined as "openshift-monitoring", when we actually install the operator, the role is being created in the "openshift-lightspeed" NS instead?

and that you think the solution(workaround) is to make this a clusterrole instead?

Reply:
need an answer/resolution on this thread